lerna.json

```diff
@@ -4,5 +4,5 @@
     "vue-components",
     "docs"
   ],
-  "version": "0.0.0"
+  "version": "1.1.0"
```

Comment:
You don't have to change this.

Reply:
we made the release with lerna. https://github.com/lerna/lerna/tree/master/commands/version#--create-release-type
and it was failing until we changed this.

Reply:
Yes, my understanding is that the version here should reflect the version number of the packages in the repository. 

> `version`: the current version of the repository.

https://github.com/lerna/lerna#lernajson

After updating this, the [`lerna version`](https://github.com/lerna/lerna/tree/master/commands/version) command worked perfectly, and none of the changes you see in 37016d2014e0eaf4d290ad1d4e742aab160c6aa2 are done by hand. I think that's quite a useful feature.

Reply:
Oh cool! Can you update the docs please? https://wmde.github.io/wikit/?path=/story/documentation-how-to-create-reusable-components-developing-components--page#releasing-a-new-component

Reply:
:+1: absolutely, we were planning on doing that!

Reply:
I will create a different pull request for this.